lib/persistable.ts

```diff
@@ -1,8 +1,12 @@
 import {SupertypeSession} from "supertype";
 type Constructable<BC> = new (...args: any[]) => BC;
 
+
 export class Persistor extends SupertypeSession {
-    static create(): Persistor | undefined {return undefined};
+    /**
+     * @TODO: was typed `Persistor` but that's weird? doesn't work need to figure out what's going on.
+     */
+    static create(): any | undefined {return undefined}; 
```

Comment:
spoke offline - but note prompting discussion = "should this be typed as persistor"